containers/docker-compose-unstable.yml

```diff
@@ -15,16 +15,32 @@ services:
       - ../../dump:/docker-entrypoint-initdb.d
       # for importing a dump
       # if you want to re-import simply put away ../../mysql/*
+  redis:
+    image: redis:latest
+    ports:
+      - '6379:6379'
```

Comment:
Do not do this, you'd be opening Redis to the network.
There shouldn't be a need to expose Redis but instead to `link` redis just like the `db` container does.